Mathlib/LinearAlgebra/BilinearForm/Properties.lean

```diff
@@ -491,6 +502,26 @@ theorem apply_dualBasis_right (B : BilinForm K V) (hB : B.Nondegenerate) (sym :
   rw [sym, apply_dualBasis_left]
 #align bilin_form.apply_dual_basis_right BilinForm.apply_dualBasis_right
 
+lemma dualBasis_dualBasis_flip (B : BilinForm K V) (hB : B.Nondegenerate) {ι}
```

Comment:
```suggestion
lemma dualBasis_dualBasis_flip {B : BilinForm K V} (hB : B.Nondegenerate) {ι}
```
This can be `simp`.

Reply:
I think `B` should be explicit following the lemmas above.